content/blog/cloud-native-infrastructure-with-kubernetes-and-pulumi/index.md

```diff
@@ -121,24 +119,28 @@ Kubernetes config updates, and your cloud infrastructure.
 
 ## What's Next
 
-Today is just the beginning of an exciting journey, with Pulumi's
-support for Kubernetes and cloud native architectures. We are excited to
-tell you about what we've been up to, and hope that real languages will
+With Pulumi's
+support for Kubernetes and cloud native architectures, we hope that real languages will
 bring you more joy, reduce toil, and ultimately lower the barriers to
 entry for using Kubernetes with real components and reuse.
 
 We can't wait to hear what you think. To learn more or give it a try,
 please check out these resources:
 
-[Instal Pulumi]({{< ref "/docs/reference/install.md" >}})
+- [EKS Quickstart]({{< ref "/docs/reference/tutorials/kubernetes/tutorial-eks" >}})
+- [GKE Quickstart]({{< ref "/docs/reference/tutorials/kubernetes/tutorial-eks" >}})
```

Comment:
Looks like the hyperlink points to EKS, not GKE.

Reply:
Good catch, thanks!